src/Makevars

```diff
@@ -20,7 +20,7 @@ $(STATLIB):
 	fi && \
 	export PATH="$(PATH):$(HOME)/.cargo/bin" && \
 	if [ "$(RPOLARS_ALL_FEATURES)" == "true" ]; then \
-		$(BUILD_COMMAND) --all-features; \
+		$(BUILD_COMMAND) --features "simd"; \
```

Comment:
To debug the new polars IPC handling, I needed threads do a lot of std.out printing. I revisiting the new pool handler it will become much easier to debug.  I have hidden the debug printing behind the feature rpolars_debug_print.

I will try to make it prettier

Reply:
Ok, thank for the update.